.teamcity/_self/projects/WPComTests.kt

```diff
@@ -107,20 +111,37 @@ fun gutenbergPlaywrightBuildType( targetDevice: String, buildUuid: String, atomi
 				// The reason for this is an inconsistent issue breaking the login in AT test sites when
 				// more than one test runs in parallel. Remove or set it to 16 after the issue is solved.
 				param("JEST_E2E_WORKERS", "1")
+
 			}
 
-			// Nightlies only make sense in the AT context for now, but we leave it here to make it easier
-			// to setup a new nightly build for simple in the future. Nightly+edge doesn't make sense,
-			// but I don't think guarding against that here is worth the additional logic, as this is
-			// a factory function and a mistake like that should be clear by checking the arguments.
 			if (nightly) {
 				param("env.GUTENBERG_NIGHTLY", "true");
+				password("GB_E2E_ANNOUNCEMENT_SLACK_API_TOKEN", "credentialsJSON:8196e9b8-cf0a-4ab5-9547-95145134f04a", display = ParameterDisplay.HIDDEN);
+				// Once we move back to #gutenberg-on-dotcom in Gutenbot, change this to `GB_ANNOUNCEMENT_SLACK_CHANNEL_ID` that points
+				// to a token that has the corresponding channel id.
+				password("GB_E2E_ANNOUNCEMENT_SLACK_CHANNEL_ID_TEST", "credentialsJSON:180d1bb6-a28e-4985-bf9a-8acba63bb90c", display = ParameterDisplay.HIDDEN);
+				text("GB_E2E_ANNOUNCEMENT_THREAD_TS", value = "", allowEmpty = true, display = ParameterDisplay.HIDDEN);
```

Comment:
**question:non-blocking** - these two lines are meant to obfuscate the Slack channel name? Not sure what the "TS" in `GB_E2E_ANNOUNCEMENT_THREAD_TS` is referring to. 

Since I don't think it's possible to view the data contained in "credentialsJSON:..." again, it might make sense to document as much as feasible in comments.

Reply:
Hi Edwin 👋🏻, thanks for the review! Yes, I was extra ~paranoid~ careful here and obfuscated the channel ID on Slack. The TS is a timestamp that's returned by the Slack message posting API endpoint and is used as a unique key to find that message and post other threaded messages to it. I'll add some comments there to clarify.